e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js

```diff
@@ -389,7 +389,9 @@ context('Amendments tasks - manual amendment tasks', () => {
     pages.tasksPage.tasks.row(4, 4).status().contains('Cannot start yet');
     pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
 
-    // TODO: DTFS2-6210 add when add decision is dependent on tasks, IF THIS IS REQUIRED.
+    cy.login(UNDERWRITER_MANAGER_1);
+    cy.visit(relative(`/case/${dealId}/underwriting`));
+    pages.underwritingPage.addAmendmentUnderwriterManagerDecisionButton().should('not.exist');
 
```

Comment:
The backed up code also had a section here where it logged in as `PIM_USER_1` and navigated to the list of all tasks.

I think this behaviour is now already covered by lines 396-400, where it logs in as a risk manager and navigates to the list of all tasks. Because the next task to complete is one for risk managers only.

So I didn't include those lines from the backed up code.